library/src/main/java/com/cottacush/android/currencyedittext/CurrencyEditText.kt

```diff
@@ -108,4 +108,16 @@ class CurrencyEditText(context: Context, attrs: AttributeSet?) : TextInputEditTe
             if (text.toString() == currencySymbolPrefix) setText("")
         }
     }
+
+
+    // currencySymbolPrefix returns null when onSelectionChanged is called first time.
+    override fun onSelectionChanged(selStart: Int, selEnd: Int) {
+        if (::currencySymbolPrefix.isInitialized.not()) return
+        val symbolLength = currencySymbolPrefix.length
```

Comment:
I think we can do away with  `&& text.toString().length >= symbolLength`. If anything, it will always force the cursor to be placed just after the `currencySymbolPrefix`.  `if (selEnd < symbolLength){}` seems enough for the check.

Reply:
Without `text.toString().length >= symbolLength` control, it crashes when edit text contains only `currencySymbolPrefix` and you focus another view like the edit text below in the example. 

`IndexOutOfBoundsException: setSpan (2 ... 2) ends beyond length 0`

Reply:
I see. onSelectionChanged is getting called when EditText about to lose focus. This makes sense then.